host_vars/ohlauer-gw/gateway.yml

```diff
@@ -18,3 +18,9 @@ tunnel_metric: 1024
 # Tunnel metric 1024 as most internet uplinks will hardly reach 40MBit/s
 # 2001:bf7:830:8300::/56 is the base prefix
 
+wireguard:
+  port_start: 51820
+  port_end: 52820
+  ipv6: 2001:bf7:830:8301::/64
+  wg_key: '/root/wg.key'
+  wg_pub: '/root/wg.key'
```

Comment:
Furthermore, please include both files in
`/group_vars/all/general.yml` / sysupgrade_preserve_custom_files